lua/lspconfig/util.lua

```diff
@@ -3,6 +3,7 @@ local validate = vim.validate
 local api = vim.api
 local lsp = vim.lsp
 local uv = vim.loop
+local nvim_nine = vim.fn.has 'nvim-0.9'
```

Comment:
```suggestion
local nvim_nine = vim.fn.has('nvim-0.9') > 0
```
(a common error)

This evaluates to true in nvim-0.10 as well as in nvim-0.9.

Actually I would suggest: instead of using `has()`, we could do a nil check on `vim.iter`, see below comments.

Reply:
just equal 1